### PR TITLE
feat(EntryCard): add size=auto to EntryCard

### PR DESCRIPTION
**Pull request by anho**
# Purpose of PR

Introduce `size=auto` to EntryCard to achieve minimal height based on the props provided. This will maintain previous behaviour and gives us the opportunity to render long lists in a more narrow fashion.

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx [hunk 1]
```diff
@@ -234,11 +237,11 @@ export class EntryCard extends Component<EntryCardPropTypes> {
                   <div className={styles.EntryCard__body}>
                     {title && this.renderTitle(title)}
                     {description &&
-                      size === 'default' &&
+                      actualSize === 'default' &&
                       this.renderDescription(description)}
                   </div>
                   {thumbnailElement &&
-                    size === 'default' &&
+                    actualSize === 'default' &&
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx:244]: As part of the story we should be able to show the thumbnail even if the card is small. People rely on images when they search their content.

**Review by gui-santos — APPROVED** [review]
🚢 

and because you added a new story,
100 points for Griffindor 🦁

**Review by burakukula — APPROVED** [review]
👏

**Comment by anho** [comment]
@gui-santos @burakukula I provided a tiny bit more info in the opening comment due to some more changes.

**Review by suevalov — APPROVED** [review]

**Pull request merged**